lib/cc/engine/checkstyle.rb

```diff
@@ -50,10 +50,25 @@ def include_paths
       end
 
       def config_path
-        if @engine_config["config"] && File.exists?(@engine_config["config"])
-          return "/code/#{@engine_config["config"]}"
+        return "/usr/src/app/config/codeclimate_checkstyle.xml" unless config
```

Comment:
Probably want `unless config_file`. A user could specify `config:` hash that doesn't include a `file:` key.